lib/sdn_epc/forwarder.ex

```diff
@@ -75,11 +77,19 @@ defmodule SdnEpc.Forwarder do
       {:send_msg_to_controller, switch_id, msg})
   end
 
+  @doc """
+  Change Forwarder mode of working. Possible options are :forwarding (default)
+  and :blocking.
+  """
+  @spec switch_mode(mode :: atom()) :: term()
+  def switch_mode(mode) do
```

Comment:
I don't think exposing the mode out of the forwarder is a good idea. I suggest `Forwarder.block()` API.